src/result.rs

```diff
@@ -120,6 +117,7 @@ impl QueryResultWrapper {
   #[napi]
   // As we don't have UUID type, we temporary use string
   pub fn get_trace_id(&self) -> Option<String> {
+    // TODO, FIXME when UUID type is implemented
```

Comment:
The proper format is:
`// FIXME: stop using strings when UUID type is implemented`

Reply:
Dedicated tools recognize `FIXME:` and `TODO:` parts, detect them and even provide syntax colouring.